control_panel_api/aws.py

```diff
@@ -1,106 +1,109 @@
 import json
 
-from django.conf import settings
-
-
-aws_api_client = settings.AWS_API_CLIENT_HANDLER
-
-
-def create_bucket(name, region, acl='private'):
-    aws_api_client('s3').create_bucket(
-        Bucket=name,
-        ACL=acl,
-        CreateBucketConfiguration={'LocationConstraint': region},
-    )
-
-
-def put_bucket_logging(name, target_bucket, target_prefix):
-    aws_api_client('s3').put_bucket_logging(
-        Bucket=name,
-        BucketLoggingStatus={
-            'LoggingEnabled': {
-                'TargetBucket': target_bucket,
-                'TargetPrefix': target_prefix
-            }
-        }
-    )
+import boto3
 
+from django.conf import settings
 
-def create_policy(name, policy_document):
-    aws_api_client("iam").create_policy(
-        PolicyName=name,
-        PolicyDocument=json.dumps(policy_document)
-    )
 
+class AWSClient(object):
 
-def delete_policy(policy_arn):
-    aws_api_client("iam").delete_policy(PolicyArn=policy_arn)
+    def __init__(self):
+        self.enabled = settings.ENABLED['write_to_cluster']
```

Comment:
Super-minor thing. I wonder if the constructor should receive this `enabled` option instead of reading a global variable directly (`settings`).

Reply:
Yep, maybe. Not really necessary, as `aws_api_client` is essentially a singleton - no other instances will be created.